src/main/java/com/minecolonies/coremod/entity/ai/citizen/cook/EntityAIWorkCook.java

```diff
@@ -214,6 +215,7 @@ private IAIState serveFoodToCitizen()
                     }
                 }
             }
+            worker.getCitizenColonyHandler().getColony().getStatisticsManager().increment(FOOD_SERVED);
```

Comment:
Shouldn't this be only when `stack.isEdible()` and hence food was actually served?
(Side note: if the stack is _not_ edible due to some mismatch or race condition, doesn't that code destroy an item?)